src/validation/validation.ts

```diff
@@ -0,0 +1,26 @@
+import { LanguageCode } from 'src/language'
+
+export enum Severity {
```

Comment:
should we call this ValidationSeverity ?

Reply:
right